include/sound/sof/info.h

```diff
@@ -25,6 +25,7 @@
 #define SOF_IPC_INFO_LOCKS		BIT(1)
 #define SOF_IPC_INFO_LOCKSV		BIT(2)
 #define SOF_IPC_INFO_GDB		BIT(3)
+#define SOF_IPC_INFO_IMR_RESTORE	BIT(4)
```

Comment:
ABI change?

Reply:
> ABI change?

It is backward compatible, e.g. all FW/driver combination works, so I think we don't need ABI bump here:
old FW + old driver, no IMR restore.
old FW + new driver, no IMR restore.
new FW + old driver, driver FW purge IPC and re-downloading, no IMR restore.
new FW + new driver, IMR restore.

Reply:
> ment MINOR version if you add backwards compatible features or
> changes. PATCH should be reset to 0.

OK, let me bump it to 3.19.0 then.